### DISCUSSION TITLE
Benefits of using this over Vidupe

### DISCUSSION THREAD

**Discussion by EldrinSMP** [theophanemayaud/video-simili-duplicate-cleaner#117] [Q&A]
Hello!

I currently use the last available release of Vidupe (v1.211 released on 11/19/2019) and it works very well. I would be interested in know what improvements have been made to the application since then, since the new version costs money to use and the old is free.  

Thanks!
--

PS: If you're charging for it, you MIGHT want to change your licensening statement on your Credits.nd page, which states

> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

**Comment by theophanemayaud** [comment]
Hi !! Thanks for taking the time to write your feedback. The main technical change was that I needed it for macOS and the original author wasn't responding, so I made a few changes so that it would compile, compiled the libraries, and wrote how I did it. The main feature changes are the automatic deletion of files ! Otherwise there are minor elements only. You can check the changelog.md file to know everything 😊

As for "free software", it is indeed free software, although for people that want the convenience, and want to support what is useful to them, I've made a precompiled version on the app stores that is paid 💰 Still it's free to modify, fix, improve, copy, reuse, and redistribute. Although for the last one I kindly ask not to share a clone, even though you're technically allowed as it is indeed free software 😅I never made it intentionally hard to compile or anything, every guide I wrote to myself to remember how to do it, is here !

**Reply by EldrinSMP** [reply]
Cool. I appreciate your reply. I especially appreciate you explaining the last part. I honestly had not thought of that and see exactly where you are coming from. I'll be getting it from the store because, honestly, Vidupe has been very crashy for me lately.

I looked at the roadmap project you have for future development. I'm looking forward to new releases, especially the "Not a duplicate" button!
